samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -165,6 +221,8 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                 props.startCallHandler({
                   displayName,
                   teamsLink,
+                  /* @conditional-compile-remove(rooms) */
```

Comment:
this startCallhandler. How I am reading this. It means we can supply multiple different locators?I think we only want to ever have one locator?